app/assets/stylesheets/components/_auth-form.scss

```
@@ -0,0 +1,62 @@
+.auth-form {
+  width: 100%;
+  max-width: rem(450px);
+  margin: auto 0;
+
+  &__wrapper {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: rem($_auth-form-pads);
```

If you agree with the previous comment, please also update this

```suggestion
    padding: $_auth-form-pads;
```

---

👍🏾 Updated in https://github.com/aetheryna/google-scraper-ruby/pull/32/commits/9f183e0fdd578cf9667fa3ab652ca19e30405663